### PR TITLE
Hide panes

### LINKED ISSUE #4124: collapsable panes should not need to listen to param changes
## Overview
Collapsable panes do not need to listen for params to collapse.

## Expected Behavior
Collapsable panes should look at params on load and collapse accordingly. Then, they shouldn't listen to param changes but rather to expand/collapse button clicks after mounted.

## Steps to Ensure no Regressions
1. collapse either tree pane (left) or inspector pane (right) or both
2. see that the pane(s) collapse(s) accordingly
3. with one or more panes collapsed, copy the url from the browser address bar
4. paste the url into a new browser window
5. see that the pane(s) collapse(s) accordingly

### PR DESCRIPTION
**Pull request by davetsay**
Resolves #4124 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/ui/layout/pane.vue [hunk 1]
```diff
@@ -78,39 +78,27 @@ export default {
     async mounted() {
         await this.$nextTick();
         // Hide tree and/or inspector pane if specified in URL
-        this.handleHideUrl();
-        this.openmct.router.on('change:params', this.handleHideUrl);
+        if (this.collapsable) {
```
Comment (kobe1104) [inline src/ui/layout/pane.vue:81]: Hi @davetsay I think we can get rid of 'collapsable' by checking this.hideParam. Only the 2 panes that has this props are collapsable.

**Review by kobe1104 — CHANGES_REQUESTED** [review]
Hi @davetsay somehow I'm not able to go to your branch so I couldn't test it on local. Does it collapse/expand the panes when you add/remove the params on the URL directly? Asking because I don't see any listeners here.

**Comment by kobe1104** [comment]
@davetsay I'm able to checkout the branch now. I noticed it requires to reload the page to collapse/expand the panes when adding the params manually on the URL.

**Review by nikhilmandlik — APPROVED** [review]
LGTM

File: src/ui/layout/pane.vue (hunk 1 above)
Comment (davetsay) [reply src/ui/layout/pane.vue:81]: changed collapsable from a prop to a computed based on hideParam prop

**Comment by davetsay** [comment]
> @davetsay I'm able to checkout the branch now. I noticed it requires to reload the page to collapse/expand the panes when adding the params manually on the URL.

This is by design. The panes are collapsed on load depending on the hide param. There is no need to listen to param change for collapsing as it is an explicit user action. Users should not be manipulating the url, only sharing it.

**Review by kobe1104 — APPROVED** [review]
Hi @davetsay as discussed: I've remove 'beforeDestroy' from pane.vue. No issues spotted other than that.

**Pull request merged**